diff --git a/thirdparty/glslang/glslang/Include/InfoSink.h b/thirdparty/glslang/glslang/Include/InfoSink.h
--- a/thirdparty/glslang/glslang/Include/InfoSink.h
+++ b/thirdparty/glslang/glslang/Include/InfoSink.h
@@ -36,9 +36,6 @@
 #define _INFOSINK_INCLUDED_
 
 #include "../Include/Common.h"
-#if __has_include(<filesystem>)
-#include <filesystem>
-#endif
 #include <cmath>
 
 namespace glslang {
@@ -107,22 +104,10 @@
         }
 
         if(loc.getFilename() == nullptr && shaderFileName != nullptr && absolute) {
-        #if defined(__cpp_lib_filesystem)
-            append(std::filesystem::absolute(shaderFileName).string());
-        #else
             append(shaderFileName);
-        #endif
         } else {
             std::string location = loc.getStringNameOrNum(false);
-            if (absolute) {
-        #if defined(__cpp_lib_filesystem)
-                append(std::filesystem::absolute(location).string());
-        #else
-                append(location);
-        #endif
-            } else {
-                append(location);
-            }
+            append(location);
         }
 
         append(locText);
